Makefile.in

```diff
@@ -228,6 +228,28 @@ config_files: ocaml-stage1-config.status
 	/bin/echo -n "( `grep \"^SHAREDLIB_CFLAGS=\" ocaml/Makefile.config \
 		| sed 's/^SHAREDLIB_CFLAGS=//'` )" > sharedlib_cflags.sexp
 
+## For targets not supported in backend, we can install the result of stage0
+.PHONY: build_upstream
+build_upstream: ocaml/configure.ac
+	rm -rf _build_upstream
+	mkdir _build_upstream
+	rsync -a $$(pwd)/ocaml/ $$(pwd)/_build_upstream
+	(cd _build_upstream && \
+	  cat ../configure_opts_stage0 | tr -d '\n' | xargs -0 ./configure -C \
+	    --prefix=@prefix@ \
+	    --disable-ocamldoc \
+	    --disable-ocamltest \
+	    --disable-debug-runtime \
+	    --disable-instrumented-runtime \
```

Comment:
Looks good, but would you mind re-enabling the debug/instrumented runtime?

Reply:
Done.